src/bin/common.rs

```diff
@@ -592,6 +604,16 @@ fn parse_config(matches: &ArgMatches<'_>) -> Result<EncoderConfig, CliError> {
     .unwrap_or_default();
 
   let mut cfg = EncoderConfig::with_speed_preset(speed);
+
+  if matches.occurrences_of("SCENE_CHANGE_DETECTION_SPEED") != 0 {
+    cfg.speed_settings.fast_scene_detection = match scene_detection_speed {
+      0 => SceneDetectionSpeed::Slow,
+      1 => SceneDetectionSpeed::Medium,
+      2 => SceneDetectionSpeed::Fast,
+      3..=u32::MAX => cfg.speed_settings.fast_scene_detection,
```

Comment:
Do we need to go to upperbound of u32, if doing that might be better to alert user by a warning that it is going to be fast mode(which is 2)? Or just restrict to 0,1,2. and throw error.